BDArmory/FX/ExplosionFX.cs

```diff
@@ -230,7 +230,7 @@ private List<BlastHitEvent> ProcessingBlastSphere()
                 //    Debug.Log("[BDArmory.ExplosionFX] SCRange: " + SCRange + "m. Normalized Direction: " + Direction.normalized.ToString("G4"));
                 //}
 
-                Ray SCRay = new Ray(Position, SCRange > Range ? (Direction.normalized * SCRange) : (Direction.normalized * Range));
+                Ray SCRay = new Ray(Position, SCRange > Range ? (Direction * SCRange) : (Direction * Range));
```

Comment:
`SCRange > Range ? (Direction * SCRange) : (Direction * Range)` is still unnecessary due to the internal normalisation as both give the same as just using `Direction`.

Reply:
Righto, forgot about that part. Resolved.